tests/test_tests.py

```diff
@@ -1,54 +1,22 @@
+from unittest import mock
 import pytest
```

Comment:
u should do not change tests from mate) it was about ur own 😅

Reply:
oopsy :D